File: Jack.Model/Model/Reuniao.vb
<Serializable()> _
Public Class Reuniao

    Public Overridable Property Codigo As Integer
    Public Overridable Property Ano As Integer
    Public Overridable Property AnoCorrente As Integer
    Public Overridable Property Data As Date

    Public Overridable ReadOnly Property DataFormated As String
        Get
            Return Data.Day.ToString("00") + "/" + Data.Month.ToString("00") + "/" + Data.Year.ToString("0000")
        End Get
    End Property

End Class

File: Jack.Model/Model/Reuniao.cs
using System;
using System.Collections.Generic;
using Jack.Library.Extensions;

namespace Jack.Model
{
    [Serializable()]
    public class Reuniao: BaseModel<Reuniao>
    {
        public Reuniao()
        {
            ano = 0;
            anoCorrente = 0;
            familia = new List<Presenca>();
            data = DateTime.Now;
        }

        private int ano { get; set; }
        private int anoCorrente { get; set; }
        private DateTime data { get; set; }
        private List<Presenca> familia { get; set; }

        public virtual int Ano
        {
            get
            {
                return ano;
            }
            set
            {
                ano = value;
            }
        }

        public virtual int AnoCorrente
        {
            get
            {
                return anoCorrente;
            }
            set
            {
                anoCorrente = value;
            }
        }

        public virtual System.DateTime Data
        {
            get
            {
                return data;
            }
            set
            {
                data = value;
            }
        }

        public virtual List<Presenca> Familia
        {
            get
            {
                return familia;
            }
            set
            {
                familia = value;
            }
        }


        public virtual string DataFormated
        {
            get
            {
                return Data.ToDateFormated();
            }
        }

    }
}
